test/Oscoin/Test/Crypto/Blockchain/Block/Helpers.hs

```diff
@@ -0,0 +1,14 @@
+module Oscoin.Test.Crypto.Blockchain.Block.Helpers where
+
+import           Oscoin.Prelude
+
+import           Oscoin.Crypto.PubKey (PublicKey)
+
+import           System.IO.Unsafe (unsafePerformIO)
+import           Test.QuickCheck
+
+-- | Generates an arbitrary beneficiary.
+defaultBeneficiary
+    :: Arbitrary (PublicKey c) => PublicKey c
+defaultBeneficiary = unsafePerformIO $ generate arbitrary
+{-# NOINLINE defaultBeneficiary #-}
```

Comment:
I have some concerns about this. It introduces randomness everywhere in the tests where we use this. The randomness is not controlled by the quickcheck seed which means we can never replay this.

I see two solutions to this problem
* Lift every function that uses `defaultBeneficiary` to `Gen` and generate the beneficiary
* Choose a fixed public key that is the same for every run. (I guess this would require another method on the crypto type class.)

Reply:
The second solution is what I was going to do with the *burn address*. The problem is currently we have no way of generating a fixed public key, unless we start from a fixed address. So the plan was to work on that in a separate PR.

You're right about the uncontrolled randomness, but since we aren't actually inspecting this value anywhere yet, or using it, it has no effect on tests at the moment. As soon as we have the burn address working, I'll remove this.